docs/cli.md

```diff
@@ -0,0 +1,687 @@
+`~` refers to your home directory.
+
+```console
+$ cd ~
+$ pwd
+/Users/ohjun
+$ ls
+Applications Pictures Desktop ...
+```
+
+### Root directory `/`
+`/` refers to the root directory. This is the top-most directory in your file system, and has no parent.
+
+```console
+$ ls /
+Applications cores sbin ...
+```
+
+### Absolute Path
+An *absolute path* starts from the root directory `/`.
+
+For example, sometimes it's useful to make sure the *exact* file is correct.
+```console
+$ /usr/local/bin/python3  # One version of Python
+$ /usr/bin/python3        # Another version of Python
+```
+
+### Relative Path
+A *relative path*: starts from the current directory.
+
+For example, running an executable.
+```console
+$ ./main.exe
+```
+
+### Glob `*`
+A *glob* is a wildcard path.  The `*` symbol matches any string.
+
+```console
+$ cp -v starter_files/* .
+'starter-files/Makefile' -> './Makefile'
+'starter-files/main_test.in' -> './main_test.in'
+'starter-files/main_test.out.correct' -> './main_test.out.correct'
+'starter-files/main_test_data.tsv' -> './main_test_data.tsv'
+'starter-files/p1_library.cpp' -> './p1_library.cpp'
+'starter-files/p1_library.h' -> './p1_library.h'
+'starter-files/stats.h' -> './stats.h'
+'starter-files/stats_public_test.cpp' -> './stats_public_test.cpp'
+'starter-files/stats_tests.cpp.starter' -> './stats_tests.cpp.starter'
+```
+
+## Tips and Tricks
+
+### `clear` <kbd>Control</kbd> + <kbd>l</kbd>
+`clear` the terminal.  Pro-tip: <kbd>Control</kbd> + <kbd>l</kbd>.  That's a lowercase L.
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
+  <source src="images/cli_vid003.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Tab complete <kbd>TAB</kbd>
+<kbd>TAB</kbd> autocompletes a file or directory name. It's faster than typing the whole thing.
+
+Type the first part of a filename, then press <kbd>TAB</kbd>.  Press again to show multiple completion options.
+```console
+$ cd ~/src/e  # Press TAB twice to see options
+eecs280/     eecs281/     eecs485/
+```
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
+  <source src="images/cli_vid004.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Previous Command <kbd>⬆</kbd>
+<kbd>⬆</kbd> shows previous commands. For example, you might repeat a compile and run step while debugging.
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
+  <source src="images/cli_vid005.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Colors and Customization
+The default terminal can look a little boring without colors or path descriptions. Fortunately, there is a way to style the terminal exactly the way you want it. Below is one example.
+
+<img src="images/cli021.png" class="invert-colors-in-dark-mode" width="768px" alt="styled terminal example"/>
```

Comment:
Screenshot changed.

Just wanted to check in on whether we want to remove the PS1 customization parts? I think it can be really helpful to show the current path on the prompt, and the terminal just looks a lot nicer to look at. What do you think?

Reply:
Marking this as resolved and moving comment to more recent thread.